Sourcery/Models/Enum.swift

```diff
@@ -51,6 +51,7 @@ class Enum: Type {
     internal(set) var rawType: String? {
         didSet {
             if let rawType = rawType {
+                hasRawType = true
```

Comment:
I'm a little confused, isn't this the same as `rawType == nil && inheritedTypes.isEmpty` ?